### ISSUE TITLE
Allow Molecule.from_file() to optionally skip files with unspecified stereochemistry

### ISSUE THREAD

**Issue by jchodera** [openforcefield/openforcefield#470]
**Is your feature request related to a problem? Please describe.**
Currently, there is no way to read _only_ the molecules with fully-specified stereochemistry from a multi-molecule SDF file. `Molecule.from_file(filename)` can either fail or load all molecules with unspecified stereochemistry.

**Describe the solution you'd like**
Adding a `skip_unspecified_stereochemistry` option to `Molecule.from_file()` would be very convenient if we want to test on a bunch of molecules. It seems very likely we will encounter files full of molecules from various sources that have unspecified stereochemistry, so dealing with these files will be streamlined if we can easily ingest the ones with fully-specified stereochemistry.

**Describe alternatives you've considered**
We would otherwise have to load all molecules with `allow_unspecified_stereo=True` and then remove molecules that have unspecified stereochemistry.

**Comment by j-wags** [comment]
Agreed, though I might clarify my thoughts on one detail: This seems reminiscent of a decision in RDKit's batch reader, where loading failures get included in the list of results as `None`. Maybe we should match that behavior, instead of skipping. Users would then get information about the loading failures, but could simply barge through with `mols = [mol for mol in mols if mol is not None`

This also got me thinking about how a "paranoid" vs "permissive" mode might work. Keywords like this might have their defaults set by a module-level or class-level variable, like `def FrozenMolecule.from_file(obj, skip_unspecified_stereochemistry=FrozenMolecule.DEFAULT_SKIP_STEREO)`

**Comment by jchodera** [comment]
All of these are good ideas!
